docs/en/tutorials/how_to_install_mmdeploy_on_jetsons.md

```diff
@@ -1,127 +1,270 @@
-## How to install mmdeploy on Jetsons
+# Build for Jetson
 
-This tutorial introduces how to install mmdeploy on Nvidia Jetson systems. It mainly introduces the installation of mmdeploy on three Jetson series boards:
+In this chapter, we introduce how to install mmdeploy on NVIDIA Jetson platforms, especially on three main Jetson series boards:
```

Comment:
I believe NVIDIA have never announced these models as "main Jetson boards".

maybe "which we have verifed on the following models" is better

Reply:
Done